libcudacxx/examples/concurrent_hash_table.cu

```diff
@@ -104,8 +105,8 @@ public:
   __host__ ~concurrent_hash_table()
   {
     thrust::for_each(thrust::device,
-                     thrust::counting_iterator<size_type>(0),
-                     thrust::counting_iterator<size_type>(capacity_),
+                     ::cuda::counting_iterator<size_type>(0),
+                     ::cuda::counting_iterator<size_type>(capacity_),
```

Comment:
This PR is replacing constant iterators, so please leave the counting iterators to a different PR.